debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -71,6 +78,12 @@
     @ConfigProperty(name = PROP_PREFIX + "null.value", defaultValue = "default")
     String nullValue;
 
+    @ConfigProperty(name = PROP_PREFIX + "exact.trimming", defaultValue = "false")
```

Comment:
These new options need documentation [here](https://github.com/debezium/debezium/blob/main/documentation/modules/ROOT/pages/operations/debezium-server.adoc#redis-stream). Could you also add a sentence as discussed in the Jira issues, that capped collections are supported but should only be used as a sink when missing events is acceptable for a particular use case?

Reply:
sure